tests/unit/TextUI/SourceMapperTest.php

```diff
@@ -24,55 +24,30 @@ final class SourceMapperTest extends TestCase
 {
     public static function provider(): Generator
     {
-        $fixtureDirectory = realpath(__DIR__ . '/../../_files/source-filter');
+        // temp remove
+        $fixtureDirectory = rtrim(self::fixturePath(''), '/');
 
         yield 'file included using file' => [
             [
-                $fixtureDirectory . DIRECTORY_SEPARATOR . 'a' . DIRECTORY_SEPARATOR . 'PrefixSuffix.php' => true,
+                self::fixturePath('a/PrefixSuffix.php') => true,
```

Comment:
I don't _think_ `DIRECTORY_SEPARATOR` is relevant on modern versions of PHP and I created `fixturePath` to avoid concatenating `$fixturesDirectory` each time.

Reply:
Considering the fact that tests fail on Windows right now ... it looks like it still might be relevant ;)

Reply:
ok, it turns out it is important here, at least for the tests.